wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -1550,14 +1550,41 @@ private function execute_describe() {
 	 */
```

Comment:
Let's add this for completeness:

```suggestion
	 * Supported syntax:
	 *
	 * UPDATE [LOW_PRIORITY] [IGNORE] table_reference
	 *    SET assignment_list
	 *    [WHERE where_condition]
	 *    [ORDER BY ...]
	 *    [LIMIT row_count]
	 *
	 * @see https://dev.mysql.com/doc/refman/8.0/en/update.html
	 */
```